stream-chat-android-ui-components/src/main/res/values/styles.xml

```diff
@@ -263,6 +263,8 @@
 
     <style name="StreamUiTheme" parent="Theme.MaterialComponents.DayNight.NoActionBar">
         <item name="streamUiValidTheme">true</item>
+        <item name="streamUiMessageListStyle">@style/StreamUi.MessageList</item>
```

Comment:
This one shouldn't be needed if we apply the fourth parameter to `obtainStyledAttributes`(default style) in all places where we are obtaining attrs